src/Symfony/Component/Form/Extension/Core/DataTransformer/ArrayToPartsTransformer.php

```diff
@@ -26,39 +26,39 @@ public function __construct(
     ) {
     }
 
-    public function transform(mixed $array): mixed
+    public function transform(mixed $value): mixed
```

Comment:
I am not convinced we should rename the arguments. It will mainly cause conflicts when merging up while not adding real value.